core/forms.py

```diff
@@ -8,26 +9,29 @@
     Hacker,
     NotificationPolicy,
     School,
+    CuratorRequest,
 )
 from django_countries.fields import CountryField
 from crispy_forms.helper import FormHelper  # type: ignore
 from crispy_forms.layout import Layout, Field, HTML, Submit, Div, Fieldset
 
 
 class HackathonForm(forms.ModelForm):
-    short_name = forms.CharField(max_length=255)
+    short_name = forms.CharField(max_length=255, required=False)
     name = forms.CharField(max_length=255)
     website = forms.URLField()
     country = forms.CharField(max_length=255)
     city = forms.CharField(max_length=255)
-    image = forms.ImageField()
+    # image = forms.ImageField()
```

Comment:
how come no image?

Reply:
before it wasnt working and idk what was wrong, I added it back and it seems to work

Reply:
If you reference the image in the field below you can't just comment it out